staging/README.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+Welcome to the staging tree!
+This tree contains experimental code that does not (yet) meet the quality
+requirements of RIOT.
```

Comment:
Maybe expand this a bit with a positive reason why the code could be in staging and not just in somebody's personal branch.